app/src/reports/VariableDisparityReport.tsx

```diff
@@ -63,32 +67,22 @@ function DisVarGeo(props: {
 
       {SUPPORTED_MADLIB_VARIABLES.includes(props.dropdownVarId) && (
         <Grid container spacing={1} justify="center">
-          <WithDatasets datasetIds={datasetIds}>
+          <WithVariables queries={queries}>
             {() => {
-              const data = percentProvider
-                .getData(
-                  datasetStore.datasets,
-                  Breakdowns.byState().andTime().andRace(true)
-                )
-                .concat(
-                  percentProvider.getData(
-                    datasetStore.datasets,
-                    Breakdowns.national().andTime().andRace(true)
-                  )
-                )
+              const geoFilteredDataset = datasetStore
+                .getVariables(geoFilteredQuery)
                 .filter(
-                  (row) => row.race_and_ethnicity !== "Not Hispanic or Latino"
+                  (row) =>
+                    !["Not Hispanic or Latino", "Total"].includes(
```

Comment:
We may want to pull out these strings or write a util for this case at some point

Reply:
Definitely agree. I had been holding off on this because I thought there was some additional thought that should go into how to handle this across datasets that have different race categories, and I hadn't put the thought in haha